source/time.tex

```diff
@@ -9372,31 +9438,32 @@
     zoned_time(sys_time<Duration>)
       -> zoned_time<common_type_t<Duration, seconds>>;
 
-  template<class TimeZonePtr, class Duration>
-    zoned_time(TimeZonePtr, sys_time<Duration>)
-      -> zoned_time<common_type_t<Duration, seconds>, TimeZonePtr>;
-
-  template<class TimeZonePtr, class Duration>
-    zoned_time(TimeZonePtr, local_time<Duration>, choose = choose::earliest)
-      -> zoned_time<common_type_t<Duration, seconds>, TimeZonePtr>;
-
-  template<class TimeZonePtr, class Duration>
-    zoned_time(TimeZonePtr, zoned_time<Duration>, choose = choose::earliest)
-      -> zoned_time<common_type_t<Duration, seconds>, TimeZonePtr>;
-
-  zoned_time(string_view) -> zoned_time<seconds>;
-
-  template<class Duration>
-    zoned_time(string_view, sys_time<Duration>)
-      -> zoned_time<common_type_t<Duration, seconds>>;
-
-  template<class Duration>
-    zoned_time(string_view, local_time<Duration>, choose = choose::earliest)
-      -> zoned_time<common_type_t<Duration, seconds>>;
-
-  template<class Duration, class TimeZonePtr, class TimeZonePtr2>
-    zoned_time(TimeZonePtr, zoned_time<Duration, TimeZonePtr2>, choose = choose::earliest)
-      -> zoned_time<common_type_t<Duration, seconds>, TimeZonePtr>;
+  template<class TimeZonePtrOrName>
+    using @\exposid{time-zone-representation}@ =
+      conditional_t<is_convertible_v<TimeZonePtrOrName, string_view>,
+                    const time_zone*,
+                    remove_cv_ref<TimeZonePtrOrName>>;          // \expos
```

Comment:
Other multi-line declarations have `\expos` on the line with the exposition-only identifier being declared.